distributed/ddp/ProcessMapping.svg

```diff
@@ -0,0 +1 @@
.859375 -0.3125q0.359375 -0.21875 0.578125 -0.640625q0.234375 -0.421875 0.234375 -0.96875q0 -0.640625 -0.328125 -1.109375q-0.328125 -0.46875 -0.90625 -0.65625q-0.578125 -0.203125 -1.671875 -0.203125l-2.65625 0l0 3.9375zm7.953125 1.359375l4.40625 -11.453125l1.640625 0l4.6875 11.453125l-1.734375 0l-1.328125 -3.46875l-4.796875 0l-1.25 3.46875l-1.625 0zm3.3125 -4.703125l3.890625 0l-1.203125 -3.171875q-0.546875 -1.453125 -0.8125 -2.375q-0.21875 1.09375 -0.609375 2.1875l-1.265625 3.359375zm8.5625 4.703125l0 -11.453125l1.515625 0l0 10.09375l5.640625 0l0 1.359375l-7.15625 0zm12.84375 0l0 -11.453125l5.078125 0q1.53125 0 2.328125 0.3125q0.796875 0.296875 1.265625 1.078125q0.484375 0.78125 0.484375 1.734375q0 1.21875 -0.796875 2.0625q-0.78125 0.828125 -2.4375 1.046875q0.609375 0.296875 0.921875 0.578125q0.65625 0.609375 1.25 1.515625l2.0 3.125l-1.90625 0l-1.515625 -2.390625q-0.671875 -1.03125 -1.109375 -1.578125q-0.421875 -0.546875 -0.765625 -0.765625q-0.328125 -0.21875 -0.6875 -0.296875q-0.25 -0.0625 -0.84375 -0.0625l-1.75 0l0 5.09375l-1.515625 0zm1.515625 -6.40625l3.25 0q1.046875 0 1.625 -0.203125q0.59375 -0.21875 0.890625 -0.6875q0.3125 -0.484375 0.3125 -1.03125q0 -0.8125 -0.59375 -1.328125q-0.59375 -0.53125 -1.859375 -0.53125l-3.625 0l0 3.78125zm8.7421875 6.40625l4.40625 -11.453125l1.640625 0l4.6875 11.453125l-1.734375 0l-1.328125 -3.46875l-4.796875 0l-1.25 3.46875l-1.625 0zm3.3125 -4.703125l3.890625 0l-1.203125 -3.171875q-0.546875 -1.453125 -0.8125 -2.375q-0.21875 1.09375 -0.609375 2.1875l-1.265625 3.359375zm8.609375 4.703125l0 -11.453125l1.5625 0l6.015625 8.984375l0 -8.984375l1.453125 0l0 11.453125l-1.5625 0l-6.015625 -9.0l0 9.0l-1.453125 0zm11.5078125 0l0 -11.453125l1.515625 0l0 5.671875l5.6875 -5.671875l2.0625 0l-4.8125 4.640625l5.015625 6.8125l-2.0 0l-4.078125 -5.796875l-1.875 1.828125l0 3.96875l-1.515625 0zm22.007812 -1.359375l0 1.359375l-7.578125 0q-0.015625 -0.515625 0.171875 -0.984375q0.28125 -0.765625 0.921875 -1.515625q0.640625 -0.75 1.84375 -1.734375q1.859375 -1.53125 2.515625 -2.421875q0.65625 -0.90625 0.65625 -1.703125q0 -0.828125 -0.59375 -1.40625q-0.59375 -0.578125 -1.5625 -0.578125q-1.015625 0 -1.625 0.609375q-0.609375 0.609375 -0.609375 1.6875l-1.453125 -0.140625q0.15625 -1.625 1.125 -2.46875q0.96875 -0.84375 2.59375 -0.84375q1.65625 0 2.609375 0.921875q0.96875 0.90625 0.96875 2.25q0 0.6875 -0.28125 1.359375q-0.28125 0.65625 -0.9375 1.390625q-0.65625 0.734375 -2.171875 2.015625q-1.265625 1.0625 -1.625 1.453125q-0.359375 0.375 -0.59375 0.75l5.625 0z" fill-rule="nonzero"/><path fill="#000000" d="m387.98566 102.88231l0 -11.453125l1.515625 0l0 10.09375l5.640625 0l0 1.359375l-7.15625 0zm8.5078125 -5.578125q0 -2.859375 1.53125 -4.46875q1.53125 -1.609375 3.953125 -1.609375q1.578125 0 2.84375 0.765625q1.28125 0.75 1.953125 2.109375q0.671875 1.34375 0.671875 3.0625q0 1.75 -0.703125 3.125q-0.703125 1.375 -2.0 2.09375q-1.28125 0.703125 -2.78125 0.703125q-1.609375 0 -2.890625 -0.78125q-1.265625 -0.796875 -1.921875 -2.140625q-0.65625 -1.359375 -0.6
```

Comment:
another option to upload a image is to post it in the comment on github, and use the github perm-link in the code.

Reply:
Thanks for the quick review! Any preference on how to handle the image? It's pretty small and sometimes I've seen the GitHub permalink be not very stable (the image just disappears).

Reply:
Hey @jlin27, what's the preferred way to upload images to docs and examples?

Reply:
Hello @mrshenli and @jlin27 - I removed the image for now from the repo and replaced it with a GitHub permalink. Please let me know if this looks OK. Thank you again!